

### Mode: GFuzz | Subdir: file(1)-test(1)-scenarios_mixed_two_test-TestA01_Mixed ###

=== RUN   TestA01_Mixed
--- FAIL: TestA01_Mixed (0.01s)
panic: send on closed channel [recovered]
	panic: send on closed channel

goroutine 6 [running]:
testing.tRunner.func1.2({0x5510a0, 0x5b1530})
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1734 +0x21c
testing.tRunner.func1()
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1737 +0x35e
panic({0x5510a0?, 0x5b1530?})
	/home/ilian/Tools/go-runtime/src/runtime/panic.go:792 +0x132
scenarios.TestA01_Mixed(0xc000003880?)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go:16 +0x70
testing.tRunner(0xc000003880, 0x585cd0)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
created by testing.(*T).Run in goroutine 1
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1851 +0x413
FAIL	scenarios	0.016s
FAIL
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
TestName:  TestA01_Mixed
Import added at line: 3
Header added at line: 10
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA01_Mixed

Exit Replay with code  30 Send on close
Exit Message: panic: send on closed channel

FAIL	scenarios	0.115s
FAIL


### Mode: GFuzzHB | Subdir: file(1)-test(1)-scenarios_mixed_two_test-TestA01_Mixed ###

=== RUN   TestA01_Mixed
--- FAIL: TestA01_Mixed (0.01s)
panic: send on closed channel [recovered]
	panic: send on closed channel

goroutine 6 [running]:
testing.tRunner.func1.2({0x5510a0, 0x5b1530})
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1734 +0x21c
testing.tRunner.func1()
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1737 +0x35e
panic({0x5510a0?, 0x5b1530?})
	/home/ilian/Tools/go-runtime/src/runtime/panic.go:792 +0x132
scenarios.TestA01_Mixed(0xc000003880?)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go:16 +0x70
testing.tRunner(0xc000003880, 0x585cd0)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
created by testing.(*T).Run in goroutine 1
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1851 +0x413
FAIL	scenarios	0.017s
FAIL
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
TestName:  TestA01_Mixed
Import added at line: 3
Header added at line: 10
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA01_Mixed

Exit Replay with code  30 Send on close
Exit Message: panic: send on closed channel

FAIL	scenarios	0.118s
FAIL
Bugreport info: 0_1,fail
Bugreport info: 0_2,fail


### Mode: Flow | Subdir: file(1)-test(1)-scenarios_mixed_two_test-TestA01_Mixed ###

=== RUN   TestA01_Mixed
--- FAIL: TestA01_Mixed (0.01s)
panic: send on closed channel [recovered]
	panic: send on closed channel

goroutine 6 [running]:
testing.tRunner.func1.2({0x5510a0, 0x5b1530})
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1734 +0x21c
testing.tRunner.func1()
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1737 +0x35e
panic({0x5510a0?, 0x5b1530?})
	/home/ilian/Tools/go-runtime/src/runtime/panic.go:792 +0x132
scenarios.TestA01_Mixed(0xc000003880?)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go:16 +0x70
testing.tRunner(0xc000003880, 0x585cd0)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
created by testing.(*T).Run in goroutine 1
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1851 +0x413
FAIL	scenarios	0.017s
FAIL
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
TestName:  TestA01_Mixed
Import added at line: 3
Header added at line: 10
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA01_Mixed

Exit Replay with code  30 Send on close
Exit Message: panic: send on closed channel

FAIL	scenarios	0.116s
FAIL
Bugreport info: 0_1,fail
Bugreport info: 0_2,fail


### Mode: GFuzzHBFlow | Subdir: file(1)-test(1)-scenarios_mixed_two_test-TestA01_Mixed ###

=== RUN   TestA01_Mixed
--- FAIL: TestA01_Mixed (0.01s)
panic: send on closed channel [recovered]
	panic: send on closed channel

goroutine 6 [running]:
testing.tRunner.func1.2({0x5510a0, 0x5b1530})
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1734 +0x21c
testing.tRunner.func1()
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1737 +0x35e
panic({0x5510a0?, 0x5b1530?})
	/home/ilian/Tools/go-runtime/src/runtime/panic.go:792 +0x132
scenarios.TestA01_Mixed(0xc000003880?)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go:16 +0x70
testing.tRunner(0xc000003880, 0x585cd0)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
created by testing.(*T).Run in goroutine 1
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1851 +0x413
FAIL	scenarios	0.018s
FAIL
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
TestName:  TestA01_Mixed
Import added at line: 3
Header added at line: 10
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA01_Mixed

Exit Replay with code  30 Send on close
Exit Message: panic: send on closed channel

FAIL	scenarios	0.117s
FAIL
Bugreport info: 0_1,fail
Bugreport info: 0_2,fail


### Mode: GoPie | Subdir: file(1)-test(1)-scenarios_mixed_two_test-TestA01_Mixed ###

FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
TestName:  TestA01_Mixed
Import added at line: 3
Header added at line: 10
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA01_Mixed
Init FuzzingReplay
Reading trace from /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/fuzzingTraces/fuzzingTrace_2
Circular Panic
panic: send on closed channel

FAIL	scenarios	70.075s
FAIL


### Mode: GoPie+ | Subdir: file(1)-test(1)-scenarios_mixed_two_test-TestA01_Mixed ###

FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
TestName:  TestA01_Mixed
Import added at line: 3
Header added at line: 10
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA01_Mixed
Init FuzzingReplay
Reading trace from /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/fuzzingTraces/fuzzingTrace_4

Exit Replay with code  30 Send on close
Exit Message: panic: send on closed channel

FAIL	scenarios	0.118s
FAIL


### Mode: GoPieHB | Subdir: file(1)-test(1)-scenarios_mixed_two_test-TestA01_Mixed ###

FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
TestName:  TestA01_Mixed
Import added at line: 3
Header added at line: 10
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_two_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA01_Mixed
Init FuzzingReplay
Reading trace from /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/fuzzingTraces/fuzzingTrace_4

Exit Replay with code  30 Send on close
Exit Message: panic: send on closed channel

FAIL	scenarios	0.119s
FAIL
Bugreport info: 0_1,fail
Bugreport info: 0_2,fail
